pom.xml

```diff
@@ -335,7 +335,7 @@
       <properties>
         <sparkbundle.version>3.5</sparkbundle.version>
         <sparkshim.artifactId>spark-sql-columnar-shims-spark35</sparkshim.artifactId>
-        <spark.version>3.5.2</spark.version>
+        <spark.version>3.5.4</spark.version>
         <iceberg.version>1.5.0</iceberg.version>
```

Comment:
Change the iceberg version here?

Reply:
Yes. However, iceberg has not released version 1.7.2 yet. Iceberg 1.7.2 have a bugfix related to Spark 3.5.4, this PR needs to wait until iceberg 1.7.2 is released.